### PR TITLE
PR: Use normpath to set network folders as cwd on Windows

### LINKED ISSUE #10785: Can't change dir to network drive in console
### What steps will reproduce the problem?

ust installed spyder 4.0. this error is coming up when i access a folder or a file. 

### Traceback
```python-traceback
Exception in comms call set_cwd:
  File "C:\Anaconda3\lib\site-packages\spyder_kernels\comms\commbase.py", line 342, in _handle_remote_call
    buffer['call_kwargs'])
  File "C:\Anaconda3\lib\site-packages\spyder_kernels\comms\commbase.py", line 353, in _remote_callback
    *call_args, **call_kwargs)
  File "C:\Anaconda3\lib\site-packages\spyder_kernels\console\kernel.py", line 352, in set_cwd
    os.chdir(dirname)
FileNotFoundError: [Errno 2] The specified path is invalid: '\\\\\\\\fs01\\\\users\\\\ora\\\\Python Scripts\\\\HS_Scanner'
WARNING:spyder.widgets.github.backend:Failed to send bug report on Github. response={'code': 401, 'json': {'message': 'Bad credentials', 'documentation_url': 'https://developer.github.com/v3'}}
WARNING:spyder.widgets.github.backend:Failed to send bug report on Github. response={'code': 401, 'json': {'message': 'Bad credentials', 'documentation_url': 'https://developer.github.com/v3'}}
```

## Versions

* Spyder version: 4.0.0rc2 
* Python version: 3.6.8
* Qt version: 5.11.2
* PyQt5 version: 5.11.3
* Operating System: Windows 10

### Dependencies

```
cloudpickle >=0.5.0          :  0.8.0 (OK)
pygments >=2.0               :  2.3.1 (OK)
qtconsole >=4.6.0            :  4.6.0 (OK)
nbconvert >=4.0              :  5.4.1 (OK)
sphinx >=0.6.6               :  1.8.5 (OK)
pylint >=0.25                :  2.3.1 (OK)
psutil >=0.3                 :  5.6.1 (OK)
qtawesome >=0.5.7            :  0.5.7 (OK)
qtpy >=1.5.0                 :  1.7.0 (OK)
pickleshare >=0.4            :  0.7.5 (OK)
zmq >=17                     :  18.0.0 (OK)
chardet >=2.0.0              :  3.0.4 (OK)
numpydoc >=0.6.0             :  0.8.0 (OK)
spyder_kernels >=1.8.0;<2.0.0:  1.8.0 (OK)
qdarkstyle >=2.7             :  2.7 (OK)
atomicwrites >=1.2.0         :  1.3.0 (OK)
diff_match_patch >=20181111  :  20181111 (OK)
watchdog                     :  None (OK)
keyring                      :  None (OK)
pexpect >=4.4.0              :  4.7.0 (OK)
pympler                      :  None (OK)
sympy >=0.7.3                :  1.3 (OK)
cython >=0.21                :  0.29.6 (OK)
IPython >=4.0                :  7.4.0 (OK)
matplotlib >=2.0.0           :  3.0.3 (OK)
pandas >=0.13.1              :  0.24.2 (OK)
numpy >=1.7                  :  1.16.2 (OK)
scipy >=0.17.0               :  1.2.1 (OK)
pyls >=0.31.0;<0.32.0        :  0.31.1 (OK)
rtree >=0.8.3                :  0.8.3 (OK)
```

### PR DESCRIPTION
**Pull request by dalthviz**
## Description of Changes

* [ ] Wrote at least one-line docstrings (for any new functions)
* [ ] Added unit test(s) covering the changes (if testable)

* [ ] Included a screenshot or animation (if affecting the UI, see [Licecap](https://www.cockos.com/licecap/))

Use `os.path.normpath` to handle backward slashes instead of escaping them with `\\` to prevent errors while setting network folders as cwd on Windows

### Issue(s) Resolved

Fixes #10785 

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: dalthviz

### CODE REVIEW

**Comment by ccordoba12** [comment]
Does this fix the issue of creating projects in network drives too?

**Comment by dalthviz** [comment]
I think so, but I will check just in case

**Comment by dalthviz** [comment]
@ccordoba12 this fixes the issue with opening projects in network folders :+1:

image

File: spyder/plugins/ipythonconsole/widgets/shell.py [hunk 1]
```diff
@@ -168,7 +169,9 @@ def set_cwd(self, dirname):
         """Set shell current working directory."""
         # Replace single for double backslashes on Windows
```
Comment (ccordoba12) [inline spyder/plugins/ipythonconsole/widgets/shell.py]: Please remove this comment because it's not necessary anymore.

**Pull request merged**